src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -907,19 +907,38 @@ QueryNetworkResult Engine::QueryNetwork(const CNNNetwork& network, const std::ma
         auto ops = clonedNetwork.getFunction()->get_ordered_ops();
         std::unordered_set<std::string> supported;
         std::unordered_set<std::string> unsupported;
+
+        auto layerIsSupported = [&](std::shared_ptr<ngraph::Node> op) {
+            std::unique_ptr<MKLDNNNode> ptr;
+            try {
+                ptr.reset(MKLDNNNode::factory().create(op, {mkldnn::engine::kind::cpu, 0}, extensionManager, fake_w_cache));
```

Comment:
Node creation is a very expensive operation. @dmitry-gorokhov Do you have more light way approach to check that operation is supported?